### DISCUSSION TITLE
[New] SEO - Supprimer les pages "zombies"

### DISCUSSION THREAD

**Discussion by ACTLEM** [cnumr/best-practices#63] [Bonnes pratiques] [answered]
La masse noire sont des urls d'un site web qui sont crawlés et qui n'ont pas raison d'être. Le problème vient souvent du sitemap qui les contient. Le risque est que ces urls soient indexées alors que les pages associées ont peu d'utilité et sont habituellement peu visités au sein du site web.
Avoir des pages zombies occasionnent les effets suivants:
- Crawl inutile des robots (Google par ex.) et donc sollicitations de ressources pour rien
- Risque d'avoir des internautes arriver sur des pages sans intérêt via Google
- Souvent oubliés ces pages peuvent devenir des pages orphelines en cas de refonte et être toujours indexées

**Comment by arnolem** [comment]
Je pense qu'il faut lier tout cela avec une bonne pratique autour du sitemap.xml (**Indiquer à Google le contenu à scanner, et quand**).
On peut également parler de **noindex** pour certains contenus et bien entendu, la suppression est la meilleure des manières 👍

**Reply by ACTLEM** [reply]
Yes, c'est l'idée.

**Comment by ACTLEM** [comment, marked as answer]
Note équipe projet: Avoir une bonne pratique autour du sitemap

Issue créée: https://github.com/cnumr/best-practices/issues/172
